Toronto Star.js

```diff
@@ -233,28 +266,76 @@ var testCases = [
 	},
 	{
 		"type": "web",
-		"url": "https://www.thestar.com/yourtoronto/education_blog/2014/03/toronto_tustee_misbehaviour_isn_t_anything_new.html",
+		"url": "https://www.thestar.com/news/canada.html",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://www.thestar.com/authors.sarrouh_maria.html",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://www.thestar.com/topic.ottawa.html",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://www.thestar.com/news/canada/2021/08/12/yukon-reports-nine-new-cases-of-covid-19-territory-has-45-active-infections.html",
 		"items": [
 			{
-				"itemType": "blogPost",
-				"title": "Toronto trustee misbehaviour isn't anything new",
-				"creators": [
-					{
-						"firstName": "Kristin",
-						"lastName": "Rushowy",
-						"creatorType": "author"
-					}
-				],
-				"date": "2014-03-18",
+				"itemType": "newspaperArticle",
+				"title": "Yukon reports nine new cases of COVID-19; territory has 45 active infections",
+				"creators": [],
+				"date": "2021-08-12",
+				"ISSN": "0319-0781",
+				"abstractNote": "WHITEHORSE - Yukon health officials are reporting nine new cases of COVID-19.",
 				"language": "en-CA",
-				"url": "https://www.thestar.com/yourtoronto/education_blog/2014/03/toronto_tustee_misbehaviour_isn_t_anything_new.html",
+				"libraryCatalog": "Toronto Star",
+				"publicationTitle": "The Toronto Star",
+				"section": "Canada",
+				"url": "https://www.thestar.com/news/canada/2021/08/12/yukon-reports-nine-new-cases-of-covid-19-territory-has-45-active-infections.html",
 				"attachments": [
 					{
-						"title": "Toronto Star Snapshot",
+						"title": "Snapshot",
 						"mimeType": "text/html"
 					}
 				],
-				"tags": [],
+				"tags": [
+					{
+						"tag": "CP_URGENCY_2"
```

Comment:
Sorry, last suggestion. Could we filter out tags that contain underscores? These clearly aren't meaningful in a Zotero library.

Reply:
Good idea! done